Week-3/5_Lottery/Lottery.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.0 <0.9.0;
```

Comment:
Prefer to use a Compiler range above 0.8.0... there are many breaking changes from 0.7.0 to 0.8.0.

Reply:
done